src/lib.rs

```diff
@@ -67,3 +84,17 @@ pub trait Watcher: Sized {
 pub fn new(tx: Sender<Event>) -> Result<RecommendedWatcher, Error> {
   Watcher::new(tx)
 }
+
+
+#[test]
+fn display_formatted_errors() {
+    let expected = "Some error";
+    assert_eq!(expected,
+               format!("{}", Error::Generic(String::from(expected))));
+    assert_eq!(
+        expected,
+        format!("{}",
+            Error::Io(io::Error::new(io::ErrorKind::Other, expected))
+            )
+        );
```

Comment:
There's some kind of indentation mistake here. Also, the rest of the code is indented 2 chars, not 4.